pkgs/applications/networking/feedreaders/gnome-feeds/default.nix

```diff
@@ -17,14 +17,15 @@
 , webkitgtk
 , python3
 }:
+
 python3.pkgs.buildPythonApplication rec {
-  pname = "feeds";
+  pname = "gfeeds";
   version = "0.16.1";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World";
-    repo = "gfeeds";
+    repo = pname;
```

Comment:
No need to change this. And I think we shouldn't cause the repo could be called anything, its name doesn't change if `pname` does.

Reply:
Also not a big fan of this.

Reply:
> Also not a big fan of this.

Removed.